Resources/Prototypes/_SpaceCats/Entities/Clothing/Shoes/boots.yml

```
@@ -0,0 +1,8 @@
+- type: entity
+  parent: ClothingShoesBootsJack
+  id: ClothingShoesBootsJackSecEnginer
+  components:
+    - type: Sprite
+      sprite: _SpaceCats/Clothing/Shoes/Boots/jackboots_sec_enginer.rsi
+    - type: Clothing
+      sprite: _SpaceCats/Clothing/Shoes/Boots/jackboots_sec_enginer.rsi
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)